docs/tutorial.md

```diff
@@ -803,12 +803,8 @@ device will only be updatable when all of them are removed.
 
 ## Debugging tips
 
-The most important rule when debugging is using the `--verbose` flag
-when running fwupd or fwupdtool. Besides that, there are many
-environment variables that allow some debug traces to be printed
-conditionally, for example: `FWUPD_PROBE_VERBOSE`,
-`FU_HID_DEVICE_VERBOSE`, `FWUPD_DEVICE_LIST_VERBOSE` and many other
-plugin-specific envvars.
+The most important rule when debugging is using the `--verbose` and
+duplicate `--verbose` flag when running fwupd or fwupdtool.
```

Comment:
This feels pretty weird to have two long arguments make a different behavior.  Can we instead do a verbosity level as an optional argument?
```
--verbose
--verbose 1
--verbose 2
```

Reply:
I think repeating arguments is pretty common in low level UNIX tools -- the other way is like `libusb -vvv` but I don't know how to do single-dash multiple args. I'll experiment with an optional argument, but it's not quite as _catchy_ imho.

Reply:
Ha, so GLib treats `-vv` the same as `-v -v` which means it works like lsusb.

Reply:
> Ha, so GLib treats -vv the same as -v -v which means it works like lsusb.

Great!